kafka-rest/src/test/java/io/confluent/kafkarest/requestlog/CustomLogTest.java

```diff
@@ -34,6 +34,8 @@ public class CustomLogTest {
 
   private String logEntry;
 
+  private static String LOG_FORMAT = "%m %uri";
```

Comment:
@msn-tldr just curious, why was the change needed so that these tests needed to test a different log format? was the old one not working because of deprecated methods on the request class? doc: https://javadoc.jetty.org/jetty-12/org/eclipse/jetty/server/Request.html

Reply:
@mattwong949 yeah pretty much, also it was easier to find Request/Response jetty 12 methods for this new log format, hence easier to fix the test with new format. The format used in test is orthogonal to the actual unit test.